docker/packaging/etc_default_vitess

```diff
@@ -0,0 +1,6 @@
+VTROOT="/vt"
+VTTOP="$VTROOT/src/vitess.io/vitess"
+GOTOP="$VTTOP/go"
+GOBIN="$VTROOT/bin"
+GOPATH="$VTROOT"
```

Comment:
I don't know if we should have golang variables in the packagesa. I think in a perfect world you don't need to have any go installed to install vitess

Reply:
It's not for Go itself, but in https://github.com/vitessio/vitess/blob/master/docker/lite/Dockerfile#L38 this comment makes them sound necessary.

`# Set up Vitess environment (just enough to run pre-built Go binaries)`